examples/player/main.rs

```diff
@@ -283,23 +285,33 @@ fn main() {
 
 #[cfg(not(target_os = "android"))]
 fn main() {
-    let args: Vec<_> = env::args().collect();
-    let (no_video, use_gl, filename) = if args.len() == 2 {
-        let fname: &str = args[1].as_ref();
-        (false, false, fname)
-    } else if args.len() == 3 {
-        if args[1] == "--gl" {
-            let fname: &str = args[2].as_ref();
-            (false, true, fname)
-        } else if args[1] == "--no-video" {
-            let fname: &str = args[2].as_ref();
-            (true, false, fname)
-        } else {
-            panic!("Usage: cargo run --bin player [[--gl]|[--no-video]] <file_path>")
-        }
-    } else {
-        panic!("Usage: cargo run --bin player [--gl] <file_path>")
-    };
+    let clap_matches = clap::App::new("Servo-media player example")
+        .setting(clap::AppSettings::DisableVersion)
+        .usage("player [--gl|--no-video] <FILE>")
+        .arg(
+            clap::Arg::with_name("gl")
+                .long("gl")
+                .display_order(1)
```

Comment:
is display_order necessary?

Reply:
No, it just reorders them in help message, bringing default `--help` flag to the bottom, looked fancier to me.

Reply:
Agreed